include.json

```diff
@@ -12,7 +12,7 @@
 		"https://danielvijge.github.io/lms_mixcloud/public.xml",
 		"https://server.vijge.net/static/squeezebox/repo.xml",
 		"https://slimscrobbler.sourceforge.net/repository.xml",
-		"https://superdatetimewu.sourceforge.net/CC_JL_repo.xml",
+		"http://jivelite.picoreplayer.org/repo.xml"
```

Comment:
Not a problem with this PR per se. But the downloadable file or path must have the version number in it. Otherwise there will be issues with cached files and not matching checksums etc.